app/models/user.rb

```diff
@@ -3,4 +3,8 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
+
+  has_many :games
+  has_many :pieces
```

Comment:
oops I see , Piece already covers it piece -> game -> user so all user needs is association with the game?